tox.ini

```diff
@@ -40,7 +40,7 @@ exclude =
     mod,
 max-line-length = 88
 max-complexity = 10
-ignore = E203, E501, W503, I201, I100
+ignore = E203, E261, E265, E501, I201, I100, W503
```

Comment:
So much is being ignored now, can you explain why for each?

Reply:
E261 requires 2 spaces before an inline comment, I see no reason for that.
E265 makes it horrible to comment a block during development, as it requires a space after a comment char.
the other ones were already there.